src/main.py

```
@@ -16,17 +16,8 @@
-def safe_extract_text(driver, by, value, default="No encontrado"):
-    try:
-        return driver.find_element(by, value).text.strip()
-    except Exception:
-        return default
-
-def safe_extract_attribute(driver, by, value, attribute, default="No encontrado"):
-    try:
-        return driver.find_element(by, value).get_attribute(attribute)
-    except Exception:
-        return default
+from src.scraper import safe_extract_text, safe_extract_attribute, scrape_vacante, scrape_candidato
```

Ruff (0.12.2)

`src.scraper.safe_extract_attribute` imported but unused

Remove unused import: `src.scraper.safe_extract_attribute`

(F401)